src/ripple/protocol/impl/STBase.cpp

```diff
@@ -36,7 +36,7 @@ STBase::STBase (SField const& n) noexcept
 }
 
 STBase&
-STBase::operator= (const STBase& t)
+STBase::operator= (const STBase& t) noexcept
```

Comment:
This `static_assert(noexcept(fName->isUseful()), "");` fires.

Reply:
Right. Fixed in c3c66cba6